engine/runtime/src/test/java/org/enso/compiler/ExecCompilerTest.java

```diff
@@ -28,6 +29,10 @@ public static void initEnsoContext() {
             Paths.get("../../distribution/component").toFile().getAbsolutePath()
         )
         .option(RuntimeOptions.STRICT_ERRORS, "false")
+        .option(
+                RuntimeOptions.LOG_LEVEL,
```

Comment:
Why are these `LOG_LEVEL` changes needed?

Reply:
Otherwise we rely on the parent logger to get the level, which is not obvious what it is. I also made sure that we no longer ignore the output in various places.

Reply:
That could be right, I wasn't able to verify it. I see no harm in specifying log level explicitly, other than being a bit more verbose. But if that's a dealbreaker for you, I can remove it.